portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,32 @@
+package com.google.sps.data;
+
+/** 
+ * The object representing a comment on Lorraine's online portfolio
+ */
+public class Comment {
+
+  private String userName;
+
+  private String comment;
```

Comment:
You can group instance variables together. Instance variables should also be final and instantiated in the constructor.

https://screenshot.googleplex.com/YkOgSgMMTDq

Reply:
get it!